packages/matter.js/src/node/server/TransactionalInteractionServer.ts

```diff
@@ -139,18 +226,29 @@ export class TransactionalInteractionServer extends InteractionServer {
             fabricFiltered: true,
             tracer: this.#tracer,
             actionType: ActionTracer.ActionType.Write,
+            endpoint,
+            root: this.#endpoint,
         }).act(writeAttribute);
     }
 
     protected override async invokeCommand(
+        path: CommandPath,
         command: CommandServer<any, any>,
         exchange: MessageExchange<MatterDevice>,
         commandFields: any,
         message: Message,
         endpoint: EndpointInterface,
         timed = false,
     ) {
-        const invokeCommand = () => super.invokeCommand(command, exchange, commandFields, message, endpoint, timed);
+        const invokeCommand = (context: ActionContext) => {
+            // TODO Change this to behavior style, right now it does minimalistic whats needed
```

Comment:
I was thinking ACL enforcement for commands would live here, we could add a method to check ACLs on the behavior but I don't think I'd wire directly into the command handlers because that would be problematic with overrides.

Reply:
I just thought you planned more because there are methods like autorizeInvoke and such when i remember. We can also do like that  and good.